nodejs/packages/layer/package.json

```diff
@@ -26,29 +26,29 @@
     "node": ">=10.0.0"
```

Comment:
Should this be `>=12.0.0`?  v10 is not supported by lambda.

Reply:
+1

Reply:
updated